lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/lib/factory.js

```diff
@@ -44,23 +61,16 @@ var pow = require( '@stdlib/math/base/special/pow' );
 * y = mgf( 2.0 );
 * // returns ~10.205
 */
-function factory( a, b, c ) {
-	var bmc;
-	var bma;
-	var cma;
-
+function factory(a, b, c) {
```

Comment:
```suggestion
function factory( a, b, c ) {
```

stdlib conventions are to use spaces within brackets. Applies here and elsewhere throughout this PR.